source/js/main.js

```diff
@@ -258,37 +258,37 @@ let main = {
     }
 
     // Fellowships single filter fellow list
-    if (document.querySelectorAll(`.single-filter-fellow-list`).length) {
-      let targets = document.querySelectorAll(`.single-filter-fellow-list`);
-
-      Array.prototype.forEach.call(targets, (target) => {
-        return ReactDOM.render(
-          <SingleFilterFellowList
-            env={env}
-            filterType={target.dataset.filterType}
-            filterOptions={target.dataset.filterOptions.split(`,`)}
-            selectedOption={target.dataset.selectedOption}
-          />, target
-        );
-      });
-    }
+    let singleFilterFellowList = Array.from(
```

Comment:
updated because it used the exact same pattern as `pulseProjectList` and changing one but not the other would have just made the code inconsistent.